pkg/resources/external_function_test.go

```diff
@@ -162,6 +131,103 @@ func TestExternalFunctionDelete(t *testing.T) {
 	})
 }
 
+func TestExternalFunctionUpdateComment(t *testing.T) {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT005: acceptance test function name should begin with TestAcc